package-lock.json

```diff
@@ -2508,6 +2508,23 @@
         }
       }
     },
+    "@ionic/storage": {
```

Comment:
Este archivo creo que no habria que subirlo. Quizas deberiamos sumarlo al .gitignore

Reply:
Son necesarios para la sincronización de los paquetes y sus modificaciones
https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore